test/ILP/fakes/fake-default-ilp-clients.ts

```diff
@@ -0,0 +1,83 @@
+import DefaultIlpClient from '../../../src/ILP/default-ilp-client'
+import {
+  FakeIlpNetworkClient,
+  FakeIlpNetworkClientResponses,
+} from './fake-ilp-network-client'
+
+/**
+ * Contains all of the fake DefaultIlpClients needed by DefaultIlpClient Tests.
+ */
+export default class FakeDefaultIlpClients {
+  /**
+   * A DefaultIlpClient with a FakeIlpNetworkClient that always succeeds.
+   */
+  public static fakeSuceedingNetworkClient = (): DefaultIlpClient => {
+    return new DefaultIlpClient(new FakeIlpNetworkClient())
+  }
+
+  /**
+   * A DefaultIlpClient with a FakeIlpNetworkClient that always fails with default errors
+   */
+  public static fakeErroringNetworkClient = (): DefaultIlpClient => {
```

Comment:
Consider if you want to just use a fake static factory method that takes in an error rather than explicitly adding a static var for every error type you may handle. 

It'll cut down on verbosity in the long run.